supervisor/resolution/evaluations/operating_system.py

```diff
@@ -6,7 +6,7 @@
 from ..const import UnsupportedReason
 from .base import EvaluateBase
 
-SUPPORTED_OS = ["Debian GNU/Linux 10 (buster)"]
+SUPPORTED_OS = ["Debian GNU/Linux 10 (buster)", "Debian GNU/Linux 11 (bullseye)"]
```

Comment:
We support only one version. I don't want run 2 debian version or support sub deps in any kind of version

Reply:
According to the ADR we have a deprecation period of 4 months.

> When a new major version of Debian is released, the previous major version is dropped, with a deprecation time of 4 months

<https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md>

Which means a 4 month overlap period.